spec/application_helper_spec.rb

```diff
@@ -0,0 +1,9 @@
+require "rails_helper"
+
+describe ApplicationHelper, type: :helper do
+  describe "#page_title" do
+    it "returns GOV.UK for a normal page" do
+      expect(helper.page_title(["Test Page"])).to eq("Test Page - GOV.UK")
```

Comment:
As it accepts a splat we shouldn't be pushing in an array. e.g it should be `helper.page_title("Test Page")`